utils_mysql.go

```diff
@@ -95,20 +96,30 @@ func generateMysqlTypes(obj map[string]map[string]string, depth int, jsonAnnotat
 			annotations = append(annotations, fmt.Sprintf("gorm:\"column:%s%s\"", key, primary))
 		}
 		if jsonAnnotation == true {
-			annotations = append(annotations, fmt.Sprintf("json:\"%s\"", key))
+			res :=Lcfirst(Case2Camel(key))
```

Comment:
For the json changes, it looks like your goal is to support different json name format right? snake case vs camel, etc? If so I believe the behavior should be added as a option the user passes in.